src/com/facebook/buck/jvm/scala/ScalaLibraryBuilder.java

```diff
@@ -51,6 +51,7 @@
         cellRoots,
         javaBuckConfig);
     this.scalaBuckConfig = scalaBuckConfig;
+    setCompileAgainstAbis(false);
```

Comment:
please add a comment explaining why this is necessary

Reply:
Added a comment to point to https://github.com/facebook/buck/issues/1386